packages/fuel-gauge/src/policies.test.ts

```diff
@@ -63,7 +63,7 @@ describe('Policies', () => {
 
     const txParams: CustomTxParams = {
       tip: 10,
-      maturity: randomNumber(1, 2),
+      maturity: 1,
```

Comment:
Is this in relation to https://github.com/FuelLabs/fuels-ts/issues/2201 ?

Reply:
I made this change because `maturity` value should not be higher than the block height https://github.com/FuelLabs/fuels-ts/pull/2037/commits/596c6770ac00fefd34c981216d30d703d3461829